packages/fiber/src/core/store.ts

```diff
@@ -161,10 +161,8 @@ const createStore = (
   }
 
   // Set color management
-  if (!linear) {
-    if (!flat) gl.toneMapping = THREE.ACESFilmicToneMapping
-    gl.outputEncoding = THREE.sRGBEncoding
-  }
+  if (!linear) gl.outputEncoding = THREE.sRGBEncoding
+  if (!flat) gl.toneMapping = THREE.ACESFilmicToneMapping
```

Comment:
Is this something we should backport to v7?

Reply:
I think I have, Could you check, I'm mobile rn

Reply:
Doesn't seem like it's been updated yet.

[![image](https://user-images.githubusercontent.com/23324155/134863902-782a76c8-77e4-4966-8291-6fcdd37adcaf.png)](https://github.com/pmndrs/react-three-fiber/blob/7a170ac8ba30e5b621f5ee594606ec4782512c04/packages/fiber/src/core/store.ts#L163-L167)

Reply:
in that case, sure, you can cherry pick the commit

Reply:
I made a PR, but I don't think it cherry-picked right when I resolved merge conflicts in the same commit.